### PR TITLE
Added definitions for react-native-modalbox

### PR DESCRIPTION
**Pull request by iRoachie**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [c] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/react-native-modalbox/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "../"
+    ],
+    "types": [],
+    "noEmit": true,
+    "forceConsistentCasingInFileNames": true,
+    "jsx": "react"
+  },
+  "files": [
+    "index.d.ts",
+    "react-native-drawer-tests.tsx"
```
Comment (sandersn) [inline types/react-native-modalbox/tsconfig.json]: this needs to be updated.

**Review by sandersn — CHANGES_REQUESTED** [review]
tsconfig has a missed update that causes Travis to fail.

File: types/react-native-modalbox/tsconfig.json (hunk 1 above)
Comment (iRoachie) [inline types/react-native-modalbox/tsconfig.json]: Oh my, sorry about that

**Review by iRoachie — COMMENTED** [review]
Changes made

**Comment by iRoachie** [comment]
@sandersn Made requested changes

**Review by sandersn — APPROVED** [review]

**Pull request merged**